libs/signed_integers/src/i128.sw

```diff
@@ -95,9 +95,7 @@ impl core::ops::Add for I128 {
     fn add(self, other: Self) -> Self {
         // subtract 1 << 63 to avoid double move
         let mut res = Self::new();
-        if (self.underlying > Self::indent()
-            || self.underlying == Self::indent())
-        {
+        if (self.underlying > Self::indent() || self.underlying == Self::indent()) {
```

Comment:
```suggestion
        if (self.underlying >= Self::indent()) {
```

Reply:
I forgot why this was done exactly but I know it was on purpose. @rostyslavtyshko should be able to explain

Reply:
OK I'll mark this as resolved then